From e26a07c44f3fd208fc26ebdf4beabb586fda9161 Mon Sep 17 00:00:00 2001
From: Naveen Saini <naveen.kumar.saini@intel.com>
Date: Mon, 17 Oct 2022 16:05:15 +0800
Subject: [PATCH 4/4] Makefile: fix libxdp.pc error

Error:
do_populate_sysroot: QA Issue: libxdp.pc failed sanity test (tmpdir) in
path ... xdp-tools/1.2.8-r0/sysroot-destdir/usr/lib/pkgconfig [pkgconfig]

Upstream-Status: Inappropriate

Signed-off-by: Naveen Saini <naveen.kumar.saini@intel.com>
---
 lib/libxdp/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/libxdp/Makefile b/lib/libxdp/Makefile
index c545d9a..1720a13 100644
--- a/lib/libxdp/Makefile
+++ b/lib/libxdp/Makefile
@@ -78,8 +78,8 @@ $(OBJDIR)/libxdp.so.$(LIBXDP_VERSION): $(SHARED_OBJS)
 		      $^ $(LDFLAGS) $(LDLIBS) -o $@
 
 $(OBJDIR)/libxdp.pc:
-	$(Q)sed -e "s|@PREFIX@|$(PREFIX)|" \
-			-e "s|@LIBDIR@|$(LIBDIR)|" \
+	$(Q)sed -e "s|@PREFIX@|$(prefix)|" \
+			-e "s|@LIBDIR@|$(libdir)|" \
 			-e "s|@VERSION@|$(TOOLS_VERSION)|" \
 			< libxdp.pc.template > $@
 
-- 
2.39.5

